app/routes/users/components/UserProfile.js

```diff
@@ -135,12 +136,21 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
 
   renderFields() {
     const { user } = this.props;
-    const fields = Object.keys(fieldTranslations).filter(field => user[field]);
+    let fields = Object.keys(fieldTranslations).filter(field => user[field]);
     const tags = fields.map(field => {
       const translation = capitalize(fieldTranslations[field]);
       return (
         <li key={field}>
-          <strong>{translation}:</strong> {user[field]}
+          {field.toLowerCase().includes('email') ? (
```

Comment:
Hmmm... This is kinda hacky and hard to understand if you are new to the code. :thinking: 

Have you thought about using InfoList?
https://github.com/webkom/lego-webapp/blob/60aae52f29b89f1f7bd23150d952df69b38ac090/app/components/InfoList/index.js#L19


Example here:
https://github.com/webkom/lego-webapp/blob/60aae52f29b89f1f7bd23150d952df69b38ac090/app/routes/joblistings/components/JoblistingDetail.js#L70

Reply:
Would be nice, but I dont think its any cleaner, since we need to validate the fields existance and have a custom list item for the field.

Reply:
You could add a new object with field/render mappings to avoid the ternary inside the jsx, something like this:
```
const getTranslation = (field) => capitalize(fieldTranslations[field]);
const fieldRender = {
  email: (field, value) => (
      <li key={field}>
         <span>
                <strong>{getTranslation(field)}:</translation>{value}
                <a href={`mailto:${value}`>{value}</a>
          </span>
     </li>
  )
};
const defaultFieldRender = (field, value) => (
  <li key={field}>
     <span><strong>{getTranslation(field)}:</translation> {value}</span>
  </li>
);
const getFieldRender = (field) => fieldRender[field] || defaultFieldRender;
```
And then change this map to:
```
const tags = fields.map(field => getFieldRender(field)(field, user[field]);
```
I don't know if it's any better though, that's up to you to decide 😁